qualtran/bloqs/qft/qft_text_book.py

```diff
@@ -56,3 +75,28 @@ def decompose_from_registers(
         if self.with_reverse:
             for i in range(self.bitsize // 2):
                 yield cirq.SWAP(q[i], q[-i - 1])
+
+    def build_call_graph(self, ssa: SympySymbolAllocator) -> Set['BloqCountT']:
+        ret: Set['BloqCountT'] = {(Hadamard(), self.bitsize)}
+        if is_symbolic(self.bitsize):
+            ret |= {
+                (
+                    PhaseGradientUnitary(self.bitsize - 1, exponent=0.5, is_controlled=True),
+                    self.bitsize,
+                )
+            }
+        else:
+            for i in range(1, cast(int, self.bitsize)):
+                ret |= {(PhaseGradientUnitary(i, exponent=0.5, is_controlled=True), 1)}
+        if self.with_reverse:
+            ret |= {(TwoBitSwap(), self.bitsize // 2)}
+        return ret
+
+
+@bloq_example
+def _qft_text_book() -> QFTTextBook:
+    qft_text_book = QFTTextBook(3)
+    return qft_text_book
```

Comment:
Please also add a symbolic bloq example and include it in the generated notebook. Make sure we are able to gate costs for the symbolic example (which should work now) 

Add tests in `qualtran/bloqs/qft/qft_text_book_test.py` that verify the gate costs for symbolic and non-symbolic example and that gate costs from decomposition match the gate costs from the bloq for the non-symbolic case